test/10-verify.spec.js

```diff
@@ -731,26 +858,13 @@ for(const [version, mockCredential] of versionedCredentials) {
             error = e;
           }
           should.not.exist(error,
-            'Should not throw error when issuing with "validUntil" in past');
-        });
-        it('should NOT verify if "validUntil" in the past', () => {
-          const credential = jsonld.clone(mockCredential);
-          credential.issuer = 'did:example:12345';
-          credential.validUntil = '2025-06-31T19:21:25Z';
-          const now = '2022-10-30T19:21:25Z';
-          let error;
-          try {
-            vc._checkCredential({credential, now});
-          } catch(e) {
-            error = e;
-          }
-          should.exist(error,
-            'Should throw error when "validUntil" in the past');
+            'Should not throw error when "issuanceDate" is valid');
```

Comment:
```suggestion
            'Should not throw error when "validUntil" is valid');
```

Reply:
this should be addressed here: https://github.com/digitalbazaar/vc/pull/158/commits/e76ef4f5f7d93a73071eed6f39bbae97cb9eff59